Documentation/FontEditor/UsingFontEditor.md

```diff
@@ -0,0 +1,123 @@
+
+### Contents of the Menu Bar
+```
+File -> New Font
+File -> Open
+File -> Save
+File -> Save As...
+File -> Quit
+
+Edit -> Undo
+Edit -> Redo
+Edit -> Cut
+Edit -> Copy
+Edit -> Paste
+Edit -> Delete
+Edit -> Select All
+Edit -> Copy as Text
+
+Go -> Previous Glyph
+Go -> Next Glyph
+Go -> Go to Glyph...
+
+View -> Preview Font
+View -> Font Metadata
+View -> Unicode Blocks
+View -> Scale -> 500% | 1000% | 1500%
+
+Help -> Contents
+Help -> About Font Editor
+```
+
+### The Toolbar
+The Toolbar contains the same functional entries as that of the Menubar and is represented as clickable icons. Hovering on each icon will display additional information listed at the status bar which further states what each icon does.
+
+### The Main Workspace
+The main workspace has three partitions:
+
+- The ***left section*** contains the glyph editor window. This window has three different zoom factors for ease of use and for your viewing comfort. They are 500%, 1000% and 1500%. You can freely switch to any zoom scale at anytime while creating your font. Don't let the size of the glyph editor window fool you. This is where you will be doing most of your task for around 95% of the time. And the task is that of shaping each glyph by left+clicking on the mouse. You can form a dot, a line, or a complete glyph just by connecting the dots. If you need to undo a certain location with a black dot, you can right+click on top of the target area and it will revert back to empty. Each grid area can hold any of the three states: 1) on or black 2) off or white 3) empty or gray. On state is displayed as black by default. Off state is displayed as white by default. Empty state is displayed as gray by default. Off state or white is relevant most specially if the font is fixed-width as it dictates the distance a glyph will have from left, center or right. Empty state or gray is important for variable-width font as it affects the overall width of the glyph. For variable width fonts, you move the glyph flushed left on the glyph editor window and remove any extra empty grids by decrementing the present column counter located right under the glyph editor window and thereby leaving only the entire glyph all by itself. Below the glyph editor is the glyph tool which contains the following: pen icon for creating glyph itself, move icon to enable the entire glyph move to top, down, left or right from within the glyph editor window. Below the glyph tool are the transform icons which are flip horizontal, flip vertical, rotate counter-clockwise 90degress and rotate clockwise 90degrees. Transform tools are most helpful when copying over existing glyphs and transforming them to form a new glyph. A becomes V, M becomes W, c becomes e, n becomes u, etcetera.
```

Comment:
Zoom scale -> zoom level
This is where you will spend about 95% of the time
90degress -> 90°

Reply:
Updated as such.